lib/ansible/constants.py

```diff
@@ -85,15 +59,6 @@ def __getitem__(self, y):
         return self._value[y]
 
 
-# Deprecated constants
-BECOME_METHODS = _DeprecatedSequenceConstant(
```

Comment:
check with awx/tower , iirc they were using this hardcoded list for their drop downs

Reply:
I've confirmed with them that they have their own list.